src/tests/flyer.test.ts

```diff
@@ -263,3 +263,25 @@ describe('deleteFlyer tests', () => {
     expect(deleteFlyerResponse).toBeNull();
   });
 });
+
+describe('editFlyer tests', () => {
+  test('flyer with ID exists with changes', async () => {
+    const flyers = await FlyerFactory.create(2);
+    await FlyerModel.insertMany(flyers);
+
+    const fetchedFlyers = await FlyerRepo.getAllFlyers();
+    const firstFlyer = fetchedFlyers[0];
+
+    const randomSlug = Math.random().toString();
+    const editFlyerResponse = await FlyerRepo.editFlyer(firstFlyer.id, randomSlug);
+    expect(editFlyerResponse.categorySlug).toStrictEqual(randomSlug);
+  });
+
+  test('flyer with ID does not exist', async () => {
+    const flyers = await FlyerFactory.create(2);
+    await FlyerModel.insertMany(flyers);
+
+    const editFlyerResponse = await FlyerRepo.editFlyer('64811792f910705ca1a981f8');
```

Comment:
This looks like a semi-valid hash -- maybe make this more clear by just passing in a nonsensical string

Reply:
I get an error saying "Argument passed in must be a single String of 12 bytes or a string of 24 hex characters" with any other String so I'll keep it as is for now.

Reply:
Imagine that the frontend calls the edit flyer mutation with a string that is not of this format. e.g., the field could fail to populate correctly and an empty string could be sent over the request. As you've discovered, this would cause the backend to crash, which ideally we want to avoid -- maybe think about adding some error handling for this.

Reply:
We have error handling on the frontend side, so I don't think it's necessary to have error handling here. I think it's actually better for an error to be thrown here though. If the mutation is given an invalid ID, then, of course, we want the frontend to know that something went wrong. That way we can update the UI accordingly to let the organizations know that something went wrong when attempting to edit the flyer.